bitnami/mongodb/templates/networkpolicy.yaml

```
@@ -0,0 +1,44 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+{{- if .Values.global.networkPolicy }}
+apiVersion: {{ .Values.global.networkPolicy.apiVersion . | default "networking.k8s.io/v1" }}
+{{- else }}
+apiVersion: networking.k8s.io/v1
+{{- end }}
```

I'm not sure if this block is something we may want to move to a helpers function. Can you add your thoughts, @juan131?

---

his should be part of `bitnami-common` chart. See this PR for instance: https://github.com/bitnami/charts/pull/2732

Then you can do sth like:

```suggestion
apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
```

---

Thanks for your input. It makes more sense doing it like that.

---

@jsemander, after double-checking, [bitnami/common](https://github.com/bitnami/charts/tree/master/bitnami/common) is not compatible with `bitnami/mongodb` yet. For this reason, we think it would be better to move the block to a helpers macro. Could you please update the PR?